core/ipc/sink/kafka/server/src/main/java/org/opennms/core/ipc/sink/kafka/server/KafkaMessageConsumerManager.java

```diff
@@ -152,30 +162,12 @@ protected void stopConsumingForModule(SinkModule<?, Message> module) throws Exce
 
     @Override
     public void afterPropertiesSet() throws Exception {
-        // Set the defaults
         kafkaConfig.clear();
-        kafkaConfig.put("group.id", SystemInfoUtils.getInstanceId());
```

Comment:
It looks like we now default to using 'OpenNMS' instead of the value from `SystemInfoUtils.getInstanceId()` as the default `group.id`.

Reply:
Looks like this still has  `SystemInfoUtils.getInstanceId` as group.id 
  https://github.com/OpenNMS/opennms/pull/2112/files/d9723e6f7ccd8f8c3f2a6fd7326821c78c6c6253#diff-fd3056f4d5972e167c7e54d80f71c92fR42

Reply:
Good point. Looks like it only affects the new code in Sentinel. Shouldn't be a problem them.

Reply:
It should be configurable for both systems. For the OpenNMS one it is defined in the system properties as before and for sentinel it can be defined in the `kafka.consumer` properties.